src/modules/fuzzy-search/look-up.repository.ts

```diff
@@ -0,0 +1,56 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { LookUp, LookUpAttributes } from './look-up.domain';
+import { InjectModel } from '@nestjs/sequelize';
+import { LookUpModel } from './look-up.model';
+import { UserAttributes } from '../user/user.attributes';
+
+export interface LookUpRepository {
+  search(
+    userUuid: UserAttributes['uuid'],
+    partialName: string,
+    offset: number,
+  ): Promise<Array<LookUpAttributes>>;
+
+  instert(entry: LookUp): Promise<void>;
+}
+
+@Injectable()
+export class SequelizeLookUpRepository implements LookUpRepository {
+  constructor(
+    @InjectModel(LookUpModel)
+    private model: typeof LookUpModel,
+  ) {}
+
+  async search(
+    userUuid: UserAttributes['uuid'],
+    partialName: string,
+    offset: number,
+  ): Promise<Array<LookUpAttributes>> {
+    const query = `
+      SELECT
+        *,
+        rank_name,
+        similarity
+      FROM
+        look_up as lu,
+        to_tsvector(lu."name") document,
+        to_tsquery('${partialName}') query,
```

Comment:
Escape all the strings using [Sequelize Replacements](https://sequelize.org/docs/v6/core-concepts/raw-queries/#replacements) please